Emby.Server.Implementations/IStartupOptions.cs

```diff
@@ -0,0 +1,55 @@
+namespace Emby.Server.Implementations.ParsedStartupOptions
+{
+    public interface IStartupOptions
+    {
+        /// <summary>
+        /// --datadir
+        /// </summary>
+        string DataDir { get; }
```

Comment:
The `Dir` ones don't need to be in the interface IMO, you can't be sure other server implementations will use them.

Reply:
Ok, I'll remove from interface file.